vscode-trace-extension/src/extension.ts

```diff
@@ -32,11 +27,12 @@ export let traceLogger: TraceExtensionLogger;
 export const traceExtensionWebviewManager: TraceExtensionWebviewManager = new TraceExtensionWebviewManager();
 export const traceServerManager: TraceServerManager = new TraceServerManager();
 
-export function activate(context: vscode.ExtensionContext): ExternalAPI {
+export async function activate(context: vscode.ExtensionContext): Promise<ExternalAPI> {
     traceLogger = new TraceExtensionLogger('Trace Extension');
 
     const resourceTypeHandler: TraceExplorerResourceTypeHandler = TraceExplorerResourceTypeHandler.getInstance();
 
+    await updateTspClientUrl();
```

Comment:
In vscode, I get a linter warning on this line, regarding the usage of `await`, but I am not sure why? Do you see this too?

"Unexpected `await` of a non-Promise (non-"Thenable") value.eslint[@typescript-eslint/await-thenable](https://typescript-eslint.io/rules/await-thenable)"